datacenter/dtr/2.1/guides/install/index.md

```diff
@@ -49,6 +49,16 @@ Where the `--ucp-node` is the hostname of the UCP node where you want to deploy
 DTR. `--ucp-insecure-tls` tells the installer to trust the certificates used
 by UCP.
 
+The installer script will prompt you for some additional information, such as: 
+  * `dtr-external-url` - This is the location that DTR can be reached via docker clients. (ie `docker login <dtr ipaddress/url>`)
```

Comment:
Our markdown parser requires an \n before list items. You can also see the `netlify` checker on this PR for a staged version of your changes. Right now it's looking like this

<img width="735" alt="screen shot 2017-02-17 at 13 23 17" src="https://cloud.githubusercontent.com/assets/1525937/23083834/69ae8d2c-f514-11e6-9c17-72f16dc936be.png">